types/jquery/jquery-tests.ts

```diff
@@ -35,12 +35,36 @@ function JQueryStatic() {
 
         // $ExpectType JQuery<HTMLElement>
         $('<p></p>');
+        
+        // $ExpectType JQuery<HTMLParagraphElement>
+        $<HTMLParagraphElement>('<p></p>');
```

Comment:
There isn't a signature that supports this. I would remove the tests that declare the type parameter inline.

The alternative is to use a type assertion:
```typescript
const $p = $('<p></p>') as JQuery<HTMLParagraphElement>
// $ExpectType JQuery<HTMLParagraphElement>
$p;
```

or create a factory variable for that type:
```typescript
const $$p = $ as JQueryStatic<HTMLParagraphElement>;
// $ExpectType JQuery<HTMLParagraphElement>
$$p('<p></p>');
```